src/improvements/tasks/eth/024-U16a-opcm-upgrade-v410-op-soneium-ink/README.md

```diff
@@ -0,0 +1,41 @@
+# 024-U16a-opcm-upgrade-v410-op-soneium-ink: Upgrade 16a: OP, Soneium, Ink on Mainnet
```

Comment:
I think these tasks can now be 023 + 024 respectively. I delete an unused task prefixed with 022.